hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/MapInfoCollector.java

```diff
@@ -77,4 +79,18 @@ private long countMapWithAtleastOneQueryCache(Node node) {
         }).count();
 
     }
+
+    private long countMapWithAtleastOneIndex(Node node) {
+        return maps.stream().filter(distributedObject -> !(node.hazelcastInstance.getMap(distributedObject.getName()).isEmpty())).count();
```

Comment:
This counts the non-empty maps, not the maps with indexes. Please refer to the reference manual about indexing: https://docs.hazelcast.org/docs/latest-dev/manual/html-single/#indexing-queries